gensim/test/test_sklearn_api.py

```diff
@@ -831,6 +831,22 @@ def testTransform(self):
         self.assertEqual(matrix.shape[0], 1)
         self.assertEqual(matrix.shape[1], self.model.size)
 
+    def testFitTransform(self):
+        numpy.random.seed(0)
```

Comment:
This is global seeding (affect on interpreter state, not only for this test), please don't use it.